src/components/inputs/react-hook-form/ag-grid-table/csv-uploader/csv-uploader.tsx

```diff
@@ -45,6 +45,7 @@ const CsvUploader: FunctionComponent<CsvUploaderProps> = ({
     validateData = (_rows) => true,
     getDataFromCsv,
     useFieldArrayOutput,
+    ...props
```

Comment:
may be it's better to add config explicitly ? 
```suggestion
    config,
    ...props
```